ouroboros-network-framework/ouroboros-network-framework.cabal

```diff
@@ -60,42 +60,41 @@ library
   build-depends:       base            >=4.12 && <4.13
                      , async           >=2.1  && <2.3
                      , bytestring      >=0.10 && <0.11
+                     , cborg           >=0.2.1 && <0.3
                      , containers      >=0.5  && <0.7
+                     , dns                       < 4.0
+                     , iproute         >= 1.7.0 && < 1.8
```

Comment:
Alignment :nerd_face:.

Reply:
I didn't mind it, but ok let's change it.